manual/freemake-video-converter/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,10 @@
+$packageArgs = @{
+  packageName   = 'freemake-video-converter'
+  installerType = 'exe'
+  silentArgs    = '/VERYSILENT /NORESTART'
+  url           = 'http://packages.chocolatey.adam.gabrys.biz/freemake-video-converter/freemake-video-converter-4.1.9.45.exe'
```

Comment:
You are actually already distributting it which is not legal unless you can provide proof from the vendor.